app/views/shared/_left_menu.html.haml

```diff
@@ -1,58 +1,7 @@
--# TODO: possible refactoring to make more clazzes specific.
--# Clazz navigation: top_node could be course, teacher, student ...
-
--# 2018-07-06 NP: When we get all the blockers fixed, navigtion can use services....
--# - unless @wide_content_layout
--#  :javascript
--#    PortalPages.renderNavigation(Portal.API_V1.getNavigation(), 'clazzes_nav')
-
+/ TODO: possible refactoring to make more clazzes specific.
+/ Clazz navigation: top_node could be course, teacher, student ...
```

Comment:
The `-#` comment style should be used here because it keeps the comments out of rendered html.